compiler/ast/src/ast_gen.rs

```diff
@@ -195,7 +195,7 @@ pub enum ExprKind<U = ()> {
         orelse: Box<Expr<U>>,
     },
     Dict {
-        keys: Vec<Expr<U>>,
+        keys: Vec<Option<Box<Expr<U>>>>,
```

Comment:
```suggestion
        keys: Vec<Option<Expr<U>>>,
```

@youknowone Can we remove `Box` or not?